hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp

```diff
@@ -58,16 +56,16 @@ HROSCognitionMaraComponentsNode::HROSCognitionMaraComponentsNode(const std::stri
     std::string motor_name = id + "state" + axis;
 
     auto subscriber = this->create_subscription<hrim_actuator_rotaryservo_msgs::msg::StateRotaryServo>(
-                              motor_name,
+                              motor_name, rclcpp::SensorDataQoS(),
                               [this, motor_name](hrim_actuator_rotaryservo_msgs::msg::StateRotaryServo::UniquePtr msg) {
                                 stateCallback(motor_name, msg->velocity, msg->position, msg->effort);
-                              },rmw_qos_profile_sensor_data);
+                              });
     motor_state_subcriptions_.push_back(subscriber);
     std::cout << "Subscribe at " << motor_name << std::endl;
 
     motor_name = id + "goal" + axis;
 
-    auto publisher_command = this->create_publisher<hrim_actuator_rotaryservo_msgs::msg::GoalRotaryServo>(motor_name, rmw_qos_profile_sensor_data);
+    auto publisher_command = this->create_publisher<hrim_actuator_rotaryservo_msgs::msg::GoalRotaryServo>(motor_name, rclcpp::SensorDataQoS(rclcpp::KeepLast(1)));
```

Comment:
You are adding `depth=1`, just warning in case its not intented.

Reply:
Thanks @nzlz ! Yes, it is intended just to follow a common standard. Do you think that we should change depth value in this case?

Reply:
Nothing against the change, I think it won't hurt.

Reply:
Nice, thank you @nzlz!